### PR TITLE
#164047053 setup django to use postgreSQL

### PR DESCRIPTION
**Pull request by kevpy**
#### What does this PR do
Set up [ah-olympians](https://github.com/andela/ah-olympians) to use postgreSQL database

#### Description of Task to be completed?
- Setup postgreSQL database engine as the default engine
- Remove SQLite3 database engine from being the default engine
- Add .env.example as an example .env file for database environment configuration  

#### How should this be manually tested?
- Pull the branch `git fetch ch-set-up-postgresql-164047053`
- Install the dependencies `pip install -r requirements.txt`
- Ensure you have a running postgreSQL database instance running.
- Create a database on postgres. 
- Create a `.env` file with the database configs as shown in the `.env.example` file. 
- Export the `.env` file with `source .env`
- Make migrations `python manage.py makemigrations` on `ah-olympians` root folder
- Run database migrations with `python manage.py migrate`
- Run the application instance with `python manage.py runserver`

#### Any background context you want to provide?
- Every Django application comes with `SQLite3` as the default database engine setup

#### What are the relevant pivotal tracker stories?
- [#164047053](https://www.pivotaltracker.com/story/show/164047053)

### CODE REVIEW

**Comment by kelvinndmo** [comment]
Good work @kevpy

**Comment by JumaKahiga** [comment]
Correct the typo in how the PR is to be manually tested `running pos(t)greSQL`, otherwise good work changing the DB.

File: authors/settings.py [hunk 1]
```diff
@@ -134,10 +138,15 @@
...
+    # 'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
-    ),
+        'rest_framework.authentication.BasicAuthentication',
+        'rest_framework.authentication.SessionAuthentication',
+    )
+
+    # 'DEFAULT_AUTHENTICATION_CLASSES': (
```
Comment (JoanNgatia) [inline authors/settings.py]: @kevpy Please revert this change and keep the JWT authentication setup. i.e remove the basic and session authentication values.
